packages/esm-service-queues-app/src/types/index.ts

```diff
@@ -429,15 +428,22 @@ export interface QueueTableCellComponentProps {
   queueEntry: QueueEntry;
 }
 
-export type QueueTableColumn = (t: TFunction) => {
-  header: string; // header of the column, MUST be unique for each column in the queue table
+export type QueueTableColumn = {
+  key: string; // key used by DataTable, MUST be unique for each column in the queue table
+  header: string; // header of the column
```

Comment:
This supports a message code right?  Might want to indicate this can be a translation string or actual column value